Ciac 15957 Extend the CheckPoint Firewall v2 integration with additional commands (#43627)
demisto/content: Packs/CheckpointFirewall/ReleaseNotes/2_3_38.md

#### Integrations

##### CheckPoint Firewall v2

> - Updated the Docker image to: *demisto/python3:3.12.13.7444307*.

Reviewer: ```suggestion
```

> - Added support for **checkpoint-nat-rule-update** command that update an existing nat rule.

Reviewer: ```suggestion
- Added support for the following commands:  
  -  ***checkpoint-nat-rule-update***, that updates an existing nat rule.
```

> - Added support for **checkpoint-network-add** command that create a new network object.

Reviewer: ```suggestion
  - ***checkpoint-network-add***, that creates a new network object.
```

> - Added support for **checkpoint-icmp-service-update** command that update an existing icmp service object.

Reviewer: ```suggestion
  - ***checkpoint-icmp-service-update***, that updates an existing icmp service object.
```

> - Added support for **checkpoint-udp-service-update** command that update an existing udp service object.

Reviewer: ```suggestion
  - ***checkpoint-udp-service-update***, that updates an existing udp service object.
```

> - Added support for **checkpoint-network-update** command that update an existing network object.

Reviewer: ```suggestion
  - ***checkpoint-network-update***, that updates an existing network object.
```

> - Added support for **checkpoint-nat-rule-get** command that gets full data for the specified nat rule.

Reviewer: ```suggestion
  - ***checkpoint-nat-rule-get***, that gets full data for the specified nat rule.
```

> - Added support for **checkpoint-tcp-service-add** command that create a new tcp service object.

Reviewer: ```suggestion
  - ***checkpoint-tcp-service-add***, that creates a new tcp service object.
```

> - Added support for **checkpoint-icmp-service-add** command that create a new icmp service object.

Reviewer: ```suggestion
  - ***checkpoint-icmp-service-add***, that creates a new icmp service object.
```

> - Added support for **checkpoint-nat-rule-list** command that retrieve a list of nat rules from the rulebase.

Reviewer: ```suggestion
  - ***checkpoint-nat-rule-list***, that retrieves a list of nat rules from the rule base.
```

> - Added support for **checkpoint-service-delete** command that delete a service object.

Reviewer: ```suggestion
  - ***checkpoint-service-delete***, that deletes a service object.
```

> - Added support for **checkpoint-network-list** command that retrieve a list of network objects.

Reviewer: ```suggestion
  - ***checkpoint-network-list***, that retrieves a list of network objects.
```

> - Added support for **checkpoint-service-get** command that gets full data for the specified service object.

Reviewer: ```suggestion
  - ***checkpoint-service-get***, that gets full data for the specified service object.
```

> - Added support for **checkpoint-service-list** command that retrieve a list of service objects. when identifier is provided, returns a single service object.

Reviewer: ```suggestion
  - ***checkpoint-service-list***, that retrieves a list of service objects. When an identifier is provided, returns a single service object.
```

> - Added support for **checkpoint-tcp-service-update** command that update an existing tcp service object.

Reviewer: ```suggestion
  - ***checkpoint-tcp-service-update***, that updates an existing tcp service object.
```

> - Added support for **checkpoint-nat-rule-delete** command that delete a nat rule.

Reviewer: ```suggestion
  - ***checkpoint-nat-rule-delete***, that deletes a nat rule.
```

> - Added support for **checkpoint-udp-service-add** command that create a new udp service object.

Reviewer: ```suggestion
  - ***checkpoint-udp-service-add***, that creates a new udp service object.
```

> - Added support for **checkpoint-network-delete** command that delete a network object.

Reviewer: ```suggestion
  - ***checkpoint-network-delete***, that deletes a network object.
```

> - Added support for **checkpoint-nat-rule-add** command that add a new nat rule.

Reviewer: ```suggestion
  - ***checkpoint-nat-rule-add***, that adds a new nat rule.
```

> - Added support for **checkpoint-network-get** command that gets full data for the specified network object.

Reviewer: ```suggestion
  - ***checkpoint-network-get***, that gets full data for the specified network object.  
- Updated the Docker image to: *demisto/python3:3.12.13.7444307*.
```
